app/css/index.css

```
@@ -74,13 +73,12 @@ svg.spinner > circle {
 }
 
 body {
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	height: 100vh;
-	margin: 0;
+	min-height: 100vh;
+    display: grid;
+    place-content: center;
```

looks like a spacing issue, just recheck manually as not sure if github will preserve this suggestion
```suggestion
	display: grid;
	place-content: center;
```